app/globals.css

```
@@ -189,3 +189,109 @@ input[type="color"]::-webkit-color-swatch {
     z-index: 40 !important;
 }
 
+.syntax-error-highlight {
+    background-color: rgba(255, 0, 0, 0.25);
+    border-bottom: 2px wavy red;
+}
+
+
+.markdown-body {
+    word-wrap: break-word;
```

Stylelint (17.9.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)